contracts/abstracts/ListingManager.sol

```diff
@@ -43,6 +44,14 @@ abstract contract ListingManager is Controlable {
     emit ListingCreated(listingId, tokenContract, tokenId, salePrice, seller);
   }
 
+  function _editListingPrice(uint256 listingId, uint256 newSalePrice) internal returns (bool success) {
+    require(_listings[listingId].salePrice == 0, 'NFT minted');
+    require (_listings[listingId].seller == msg.sender, 'Message Sender is not the seller');
```

Comment:
Add the name of the contract where the error originate in the error message like
```text
ListingManager: Message Sender is not the seller
```